test/utilities.js

```diff
@@ -0,0 +1,29 @@
+import { Application } from 'spectron';
+import { should, use } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import electron from 'electron';
+import path from 'path';
+import LoginScreen from './src/LoginScreen';
+import Base from './src/Base';
+
+global.before(() => {
+  should();
+  use(chaiAsPromised);
```

Comment:
Would you mind adding a comment here to explain why `chaiAsPromised` is necessary  here?

Reply:
this won't be here if we use jest i think.